libsrc/apple2/mou/a2.stdmou.s

```diff
@@ -133,8 +133,8 @@ next:   inc     ptr1+1
         bcc     :+
 
         ; Mouse firmware not found
-        lda     #<MOUSE_ERR_NO_DEVICE
-        ldx     #>MOUSE_ERR_NO_DEVICE
+        lda     #MOUSE_ERR_NO_DEVICE
+        ldx     #>$0000
```

Comment:
Why did you change this (and the other similar ones)? Looks OK to me. Your change doesn't look nicer (worse in my eyes), and it breaks if MOUSE_ERR_NO_DEVICE is >= 0x100.

Reply:
Please look at our long discussion, MOUSE_ERR_NO_DEVICE >= 0x100 is impossible because the return value is char.

Reply:
Ok. (I don't remember that specific discussion right now.)
But "#>$0000" for just "0" looks really ugly to me.

Reply:
This is an intended stumbling block because it isn't a high byte of an int.